### ISSUE TITLE
VIRTUAL_ENV_DISABLE_PROMPT is not honored

### ISSUE THREAD

**Issue by desbma** [magicmonty/bash-git-prompt#122]
If VIRTUAL_ENV_DISABLE_PROMPT is set, the prompt is prefixed with Python virtual env name, unlike [the expected behavior](https://hg.python.org/cpython/file/e855bde79795/Lib/venv/scripts/posix/activate#l55) when sourcing venv scripts without bash-git-prompt.

Setting GIT_PROMPT_VIRTUALENV to an empty string does not work because of [that line](https://github.com/magicmonty/bash-git-prompt/blob/350d6c2a2ecd4fee2906e2a49fc0067fd1870d62/themes/Default.bgptheme#L66) : 
`if [[ -z ${GIT_PROMPT_VIRTUALENV} ]]; then GIT_PROMPT_VIRTUALENV="(${Blue}_VIRTUALENV_${ResetColor}) "; fi`

A workaround is to comment the line.

**Comment by ogr3** [comment]
To honor VIRTUAL_ENV_DISABLE_PROMPT you could test for it here:

```
if [[ -n "$VIRTUAL_ENV" && -z "${VIRTUAL_ENV_DISABLE_PROMPT-}" ]]; then
```

But if CONDA_DEFAULT_ENV should listen to this setting as well you should move the test up a level.

```
  local ps=""
    if [[ -z "${VIRTUAL_ENV_DISABLE_PROMPT-}" ]]; then
      if [[ -n "$VIRTUAL_ENV" ]]; then
        VENV=$(basename "${VIRTUAL_ENV}")
        ps="${ps}${GIT_PROMPT_VIRTUALENV//_VIRTUALENV_/${VENV}}"
      fi
      if [[ -n "$CONDA_DEFAULT_ENV" ]]; then
        VENV=$(basename "${CONDA_DEFAULT_ENV}")
        ps="${ps}${GIT_PROMPT_VIRTUALENV//_VIRTUALENV_/${VENV}}"
      fi
    fi
```

The test for ```VIRTUAL_ENV_DISABLE_PROMPT``` is taken directly from a virtual environment created with virtualenv.

**Comment by ogr3** [comment]
Should be fixed in #180

**Comment by desbma** [comment]
I can confirm it's fixed, thanks.
